testfiles/AbsorptionChiller.idf

```diff
@@ -108,7 +108,9 @@
   Timestep,6;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
-    1.0,                     !- Temperature Capacity Multiplier
+    ,      					 !- Name
```

Comment:
Corrected alignment using spaces for all example files. Also added name as the new idd has name as required field.